### DISCUSSION TITLE
Please enable the Virtual Machine Platform Windows feature and ensure virtualization is enabled in the BIOS. For information please visit https://aka.ms/enablevirtualization

### DISCUSSION THREAD

**Discussion by Hrishikesh-Harsh** [microsoft/WSL#10166] [General]
- I know a similar issue already exists but my case is a bit different hence this new Issue.
- Laptop was away for repairs for like 14-15 days and after it did come back everything worked well except for the WSL feature.
- Prior to that WSL worked really well.
- When I open Ubuntu app, it shows this message "Please enable the Virtual Machine Platform Windows feature and ensure virtualization is enabled in the BIOS.
For information please visit https://aka.ms/enablevirtualization" instead of the Ubuntu Terminal
- Things I've done till now to try and repair this from bits and scraps of other online forums' solutions:
   - Uninstalled and Reinstalled WSL
   - Disabled and Re-enabled WSL feature in "Turn Windows Features Off/On" settings
   - Disabled and Re-enabled Virtual Machine Platform option in "Turn Windows Features Off/On" settings
   - Tried to check for Hyper-V feature (which is the most frequent thing mentioned in all online forums); and I found no Hyper-V in 
      the "Turn Windows Features Off/On" (albeit there was Windows Hypervisor Platform that was enabled)
   - Manually ran a script to install/introduce this Hyper-V feature in "Turn Windows Features Off/On" and then enabled it
   - Disabled both Windows Hypervisor Platform and Hyper-V 
   - Most importantly, checked and verified that Virtualization is enabled in the BIOS setup
   
My machine is a Dell Inspiron 15 3000 and I'm running Windows 11. Could the issue have arisen due to a hardware fault (during the repairs). Whatever it is, I'm at my wit's end and it would be great if someone here could kindly help me out.

**Comment by Biswa96** [comment]
Does `bcdedit` command show `hypervisorlaunchtype Auto` ?

**Comment by Hrishikesh-Harsh** [comment]
> 

Currently it shows hypervisorlaunchtype Off

**Comment by Biswa96** [comment]
Try to run this command as administrator `bcdedit /set {current} hypervisorlaunchtype auto` > reboot > relaunch WSL2.

**Comment by Hrishikesh-Harsh** [comment]
> Try to run this command as administrator `bcdedit /set {current} hypervisorlaunchtype auto` > reboot > relaunch WSL2.

Ok I tried this and it still won't work. Same message on opening Ubuntu Terminal

**Comment by DoctorWho8** [comment]
What does the firmware console for your machine say? I've got a later model here, a 5000 series and needed to go into it to enable those features.

**Comment by Hrishikesh-Harsh** [comment]
> What does the firmware console for your machine say? I've got a later model here, a 5000 series and needed to go into it to enable those features.

You mean the BIOS Setup with the UEFI Firmware Settings? If yes, then I've double checked there. The Virtualization is enabled.

**Comment by Hrishikesh-Harsh** [comment]
Any follow-up/updates for this?

**Comment by whoisbyel** [comment]
The same mensage error to me, idk what i have to do !! mine ryzen 7 , geforce gtx
win 11 64x
